rdrf/rdrf/templatetags/get_translate_value.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+from rdrf.models.definition.models import Registry
+
+register = template.Library()
+
+
+@register.filter
+def get_translate_value(registry):
+    if registry:
+        return 'no' if registry.has_feature('no_translation') else 'yes'
```

Comment:
Should be added to:

https://github.com/eresearchqut/trrf/blob/9f51c60e4c3cd27d27665c513d440882b1b3dee9/rdrf/rdrf/helpers/registry_features.py#L1-L22